packages/credential-ld/src/ld-credential-module.ts

```diff
@@ -151,6 +151,12 @@ export class LdCredentialModule {
       documentLoader: this.getDocumentLoader(context, fetchRemoteContexts),
       purpose: new AssertionProofPurpose(),
       compactProof: false,
+      checkStatus: async ({ credential }: any) => {
+        const result = await context.agent.checkCredentialStatus({
```

Comment:
This will fail if the status plugin is not installed.
We'll have to find a way to check first.

Reply:
Fair enough :D Added a check. Please take a look at the solution.